tla/consistency/MultiNode.tla

```diff
@@ -13,7 +13,7 @@ ViewWithAllCommitted ==
 \* Simulates leader election by rolling back some number of uncommitted transactions and updating view
 TruncateLedgerAction ==
     /\ \E view \in ViewWithAllCommitted:
-        /\ \E i \in (CommitSeqNum + 1)..Len(ledgerBranches[view]) :
```

Comment:
This is what's causing the refinement failure, because it disallows a transition that effectively creates an empty term.

Reply:
yes, that was just a mistake.